tests/integration/test_sqlalchemy_integration.py

```diff
@@ -368,3 +379,45 @@ def test_get_table_comment(trino_connection):
         assert actual['text'] is None
     finally:
         metadata.drop_all(engine)
+
+
+@pytest.mark.parametrize('trino_connection', ['memory'], indirect=True)
```

Comment:
Instead of overriding the schema for any test you can just do 

```suggestion
@pytest.mark.parametrize('trino_connection', ['memory/test'], indirect=True)
```

That way you can get rid of the `@patch` in the next test as there won't be a default scheme set.